### PR TITLE
[docs] Add SauceLabs configuration snippet

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-sauce-labs.adoc [hunk 1]
```diff
@@ -110,3 +110,14 @@ host patterns to avoid sending of status information by SauceConnect to SauceLab
...
+
+.Configuration snippet to run tests in Chrome browser
+[source,json,subs=attributes+, source-highlighter+]
+----
+configuration.profiles=saucelabs/web,web/desktop/chrome
+
+selenium.grid.host=ondemand.us-west-1.saucelabs.com
+selenium.grid.username=secretsauce
+selenium.grid.password=87380d3afd8fd8aed55a
+saucelabs.data-center=US
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-sauce-labs.adoc:122]: no such data cebter

File: docs/modules/plugins/pages/plugin-sauce-labs.adoc [hunk 2]
```diff
@@ -110,3 +110,14 @@ host patterns to avoid sending of status information by SauceConnect to SauceLab
 a tunnel via proxy used by tests.
 
 |===
+
+.Configuration snippet to run tests in Chrome browser
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-sauce-labs.adoc:114]:
```suggestion
.Configuration snippet to run tests in Chrome browser on Sauce Labs US West data center
```

File: docs/modules/plugins/pages/plugin-sauce-labs.adoc [hunk 3]
```diff
@@ -110,3 +110,14 @@ host patterns to avoid sending of status information by SauceConnect to SauceLab
 a tunnel via proxy used by tests.
 
 |===
+
+.Configuration snippet to run tests in Chrome browser on Sauce Labs US West data center
+[source,json,subs=attributes+, source-highlighter+]
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-sauce-labs.adoc:115]:
```suggestion
[source,properties,subs=attributes+, source-highlighter+]
```

Comment (valfirst) [reply docs/modules/plugins/pages/plugin-sauce-labs.adoc:115]: and why is `subs=attributes+, source-highlighter+` needed?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/plugins/pages/plugin-sauce-labs.adoc (hunk 3 above)
Comment (uarlouski) [reply docs/modules/plugins/pages/plugin-sauce-labs.adoc]: copy paste :)

**Review by valfirst — APPROVED** [review]

**Pull request merged**